src/com/tekion/dtos/Match.java

```diff
@@ -0,0 +1,176 @@
+package com.tekion.dtos;
```

Comment:
dto no need to do dtos. packaging name is important to be consistent. we can discuss the implications in our next call.